libblkid/src/devno.c

```diff
@@ -52,16 +53,13 @@ static char *blkid_strconcat(const char *a, const char *b, const char *c)
 	if (!res)
 		return NULL;
 	if (al) {
-		memcpy(p, a, al);
```

Comment:
The braces on these blocks could now be dropped

Reply:
yeah. will do.